src/f32/vec4_f32.rs

```diff
@@ -550,13 +550,18 @@ impl Distribution<Vec4> for Standard {
     }
 }
 
+/// A 4-dimensional vector mask.
+///
+/// This type is typically created by comparison methods on `Vec4`.  It is
+/// essentially a vector of four boolean values.
```

Comment:
It might be worth mentioning that it's stored using four `u32` values. True is always `0xffffffff` and false is `0x0`. The scalar implementation also uses `u32` also so that size and alignment and values will match the SIMD version.

Reply:
Are those semantics part of the API promise here?  If so, then I agree we should add it.  But I don't see where it's exposed at the moment.

Reply:
I suppose the fact that internally `0xffffffff` is true isn't part of the public API. The size and the alignment is part of the public interface though.